DAO.sol

```diff
@@ -728,10 +736,10 @@ contract DAO is DAOInterface, Token, TokenSale {
     }
 
 
-    function addAllowedAddress(address _recipient) noEther external returns (bool _success) {
+    function changeAllowedRecipients(address _recipient, bool _allowed) noEther external returns (bool _success) {
```

Comment:
I think from a UI perspective having 2 different functions may be more beneficial.